### PR TITLE
Website updates

### PR DESCRIPTION
**Pull request by hu3man**

### CODE REVIEW

**Review by russellmorton — APPROVED** [review]

File: taqueria-plugin-contract-types/Readme.md [hunk 1]
```diff
@@ -1,14 +1,62 @@
...
+#### Parameters
+
+| parameter     |  required  | description                                        |       
+|:-------------:|:-----------|----------------------------------------------------|
+| typeOutputDir | no         | The output directory for the `.ts` files generated |
+
+#### CLI Aliases
+
+The following aliases are interchangable with `generate types`
+- `gen types`
```
Comment (alexzbusko) [inline taqueria-plugin-contract-types/Readme.md:49]: This should just be `gen`, the output from the taq help is `[aliases: gen, gentypes]`

File: website/docs/plugins/plugin-flextesa.md [hunk 2]
```diff
@@ -96,18 +96,18 @@ sandboxName : { sandboxConfigObject }
...
 
-#### 'protocol
+#### 'protocol'
 
 A string value which accepts valid Tezos protocol hashes. This value will configure the sandbox to run a particular version of the Tezos network which can be used for testing upcoming network changes
 
 Currently availble protocols are:
 - Hangzhou `PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx`
-- Ithica2  `Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A`
+- Ithica   `Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A`
```
Comment (alexzbusko) [inline website/docs/plugins/plugin-flextesa.md:110]: Ithaca

File: website/docs/plugins/plugin-typescript-generator.md [hunk 3]
```diff
@@ -51,15 +51,15 @@ taq generate types [typeOutputDir]
 
 The following aliases are interchangable with `generate types`
 - `gen types`
```
Comment (alexzbusko) [inline website/docs/plugins/plugin-typescript-generator.md:53]: This should also just be `gen`

File: website/docs/tutorials/scaffoldTutorial.mdx [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+
+### Building and Deploying the Taqueria Project
+
+If you inspect the scripts in `/taqueria/package.json`, you will see the following actions are executed when the setup command is run:
+1. Build the `/taqueria` NPM project: `npm run build` - installs npm dependencies, including Taqueria plugins
+2. Compile the ligo contract: `taq compile` - uses the LIGO Taqueria plugin to compile the smart contract located in `/taqueria/contracts`, outputing compiled Michelson `.tz` files in the `/artifacts` directory
+3. Generates TS types: `taq generate types` - uses the Taqueria Contract Types plugin to generate TS types from the compiled smart contract, and install them into the React app (`/app/types`) for use with Taquito in the frontend
+
+#### Compiling the Smart Contract
+
```
Comment (alexzbusko) [inline website/docs/tutorials/scaffoldTutorial.mdx:202]: Are these other sections going to be added in the future?

File: website/docusaurus.config.js [hunk 5]
```diff
@@ -210,14 +210,14 @@ const config = {
...
               },
-              {
-                html: `
-									<a class='footerButton' href='https://github.com/ecadlabs/taqueria'>
-										<img class='footerGihubLogoButton' src="/img/githubSVG.svg" alt="" />
-										GITHUB
-									</a>
-								  `
-              },
+              // {
```
Comment (alexzbusko) [inline website/docusaurus.config.js:213]: Should this be removed as opposed to commented out?

File: website/docs/tutorials/scaffoldTutorial.mdx (hunk 4 above)
Comment (hu3man) [reply website/docs/tutorials/scaffoldTutorial.mdx:202]: Yes, this doc is blocked until the state API is implemented as in order to use the generated types with real contract addresses, we need to be able to store the address and pass it into the front end app 

This file actually shouldn't be in this PR as it's covered in #640

File: website/docusaurus.config.js (hunk 5 above)
Comment (hu3man) [reply website/docusaurus.config.js:213]: Yes, I think this is finalized now so I'll remove

**Review by alexzbusko — APPROVED** [review]

**Pull request merged**